src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
 		)
 	);
 
+	$debug_info['buddypress-constants'] = array(
+		'label'       => __( 'BuddyPress Constants', 'buddypress' ),
+		'description' => __( 'These settings can alter where and how parts of BuddyPress are loaded or works.', 'buddypress' ),
+		'fields'      => array(
+			'BP_VERSION' => array(
+				'label' => 'BP_VERSION',
+				'value' => BP_VERSION,
+			),
+			'BP_DB_VERSION' => array(
+				'label' => 'BP_DB_VERSION',
+				'value' => BP_DB_VERSION,
+			),
+			'BP_REQUIRED_PHP_VERSION' => array(
+				'label' => 'BP_REQUIRED_PHP_VERSION',
+				'value' => BP_REQUIRED_PHP_VERSION,
+			),
+			'BP_PLUGIN_DIR' => array(
+				'label' => 'BP_PLUGIN_DIR',
+				'value' => BP_PLUGIN_DIR,
+			),
+			'BP_PLUGIN_URL' => array(
+				'label' => 'BP_PLUGIN_URL',
+				'value' => BP_PLUGIN_URL,
+			),
```

Comment:
I think we should only keep constants that can be used to customize BuddyPress or find a way to inform some are read-only.